localization/english/wc_portraits_l_english.yml

```
@@ -29,6 +29,11 @@
  brown_orcish:0 "Brown Orcish"
  pale_orcish:0 "Pale Orcish"
  
+ vrykul_ethnicity:0 "Normal Vrykul"
```

```suggestion
 vrykul_ethnicity:0 "Northern Vrykul"
```
I know they all are northern, but normal doesn't sound immersive.